EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php

```diff
@@ -32,43 +19,4 @@ class EWSType_FindMailboxStatisticsByKeywordsResponseMessageType extends EWSType
      * @var EWSType_MailboxStatisticsSearchResultType
      */
     public $MailboxStatisticsSearchResult;
-
-    /**
-     * Provides a text description of the status of the response.
-     *
-     * @since Exchange 2013
```

Comment:
Yeah I could do something like that in a separate PR that gets merged in before this one.